state-chain/runtime/src/chainflip/bitcoin_elections.rs

```diff
@@ -319,6 +335,10 @@ impls! {
 		}
 	}
 
+	Hook<HookTypeFor<Self, ElectionTrackerEventHook>> {
+		fn run(&mut self, _: <HookTypeFor<TypesFor<BitcoinVaultDepositWitnessing>, ElectionTrackerEventHook> as HookType>::Input) -> () {}
+	}
```

Comment:
Instead of implementing an empty `run()` function for the "debug hooks", we can use the `EmptyHook` struct. That is, in the `BWProcessorTypes` we can do
```rust
		type ElectionTrackerEventHook = EmptyHook;
```